bitsandbytes/utils.py

```diff
@@ -38,6 +38,16 @@ def outlier_hook(module, input):
             hook.remove()
 
 
+# convert btw standard 4-bit compression format and ipex compression format
+def reverse_4bit_compress_format(weight):
```

Comment:
I think we should mark this as a private/internal API.

```suggestion
def _reverse_4bit_compress_format(weight: torch.Tensor):
```